### ISSUE TITLE
Add support for creating new Users/Logins

### ISSUE THREAD

**Issue by ob-mbilunadev** [microsoft/azuredatastudio#21818]
~~**Is your feature request related to a problem? Please describe.**~~
~~There is no way to manage users in Azure Data Studio through the UI, I have to open up SSMS to do it.~~

~~**Describe the solution or feature you'd like**~~
~~The Database Administration Tool Extensions for Windows uses the SSMS dialog to manage server details, can this be used to manage users & logins as well?~~

~~**Describe alternatives you've considered**~~
~~I have searched for addons but can't find one.~~

Currently to create new objects such as users/logins requires manually creating and executing the CREATE statements. There should be support for creating them using a UI or similar experience similar to SSMS.

**Comment by Charles-Gagnon** [comment]
Yes it can - if you right click on a user or login with that extension installed it should give you the "Properties" menu item which will open up the SSMS dialog. Is this not working for you?

**Comment by ob-mbilunadev** [comment]
That functionality is working as expected, the issue I have is for adding new users.

In SSMS, you can right click on the Logins folder and add a new user. Would this be easy to implement in ADF?

**Comment by Charles-Gagnon** [comment]
Ah, gotcha. I updated the title & description to make that more clear - let me know if there's anything else missing from that otherwise we'll get this triaged ASAP!
